app/helpers/application_helper.rb

```diff
@@ -16,8 +16,8 @@ def markup(text)
     result[:output].to_s.html_safe
   end
 
-  def render_view_hooks(partial, locals: {})
-    Dradis::Plugins::with_feature(:addon).sort_by(&:plugin_description).each do |plugin|
+  def render_view_hooks(partial, feature = :addon, locals: {})
```

Comment:
@aapomm We've added this functionality in Pro during the permissions cycle https://github.com/secroots/dradis-pro/blob/develop/app/helpers/application_helper.rb#L23.

Can you take care of the merge conflicts once this is merged?

Reply:
I missed that one. Thanks!